Header header

uint16 height
uint16 width

uint8 detected_objects
uint8 foveation_level
sensor_msgs/Image rgb_image

FoveatedImages[] foveated_images_groups